### ISSUE TITLE
QA Nav: Mobile user account area missing C2F access

### ISSUE THREAD

**Issue by melyndav** [JoinColony/colonyCDapp#3507]
## Description

On mobile, the old user account tabs are still showing as a form of navigation, however the C2F tab is not showing in the tabs meaning I can not access this feature at all. 

<img width="441" alt="image" src="https://github.com/user-attachments/assets/4e8be35c-40db-4bcf-a85c-6a35411cb070">

The expected outcome here is for the C2F tab to be added to the mobile along with the other user account tabs to enable users to access the page.

**Comment by rumzledz** [comment]
A quick recap for anyone else reading this issue -- the C2F tab item on mobile view is protected behind a feature flag. So this tab item will only be visible if the C2F feature flag is enabled 👌

**Comment by rdig** [comment]
@rumzledz in that case we can close this issue no ?

**Comment by mmioana** [comment]
@rdig yes, I think this one can be closed as I didn't manage to reproduce it neither locally or on QA

Checked the feature flags settings from Posthog for both local and QA env and all users should have this feature visible
